### PR TITLE
Hydroflow lamport clock and shopping cart examples, Applied'23 version

### PR DESCRIPTION
**Pull request by MingweiSamuel**

### CODE REVIEW

File: hydroflow/examples/shopping/flows/client_state_flow.rs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        source_iter(shopping_ssiv)
+          -> group_by(SSIV_BOT, ssiv_merge)
+          -> map(|pair| (pair, remote_addr)) -> dest_sink_serde(carts_out);
+        source_stream_serde(carts_in) -> map(Result::unwrap) -> map(|((client, cart), _a): ((usize, SealedSetOfIndexedValues<Request>), _)| (client, cart))
+          -> [0]lookup_class;
+        source_iter(client_class) -> [1]lookup_class;
+        lookup_class = join()
+          -> map(|(client, (li, class))| ((client, class), li))
+          -> map(|m| (m, out_addr)) -> dest_sink_serde(out);
+          // -> for_each(|x| println!("{:?}\n", x));
```
Comment (MingweiSamuel) [inline hydroflow/examples/shopping/flows/client_state_flow.rs:45]: remove?

File: hydroflow/examples/shopping/flows/server_state_flow.rs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        // Networked: Server-Side State
+        source_iter(shopping_ssiv)
+          -> map(|pair| (pair, remote_addr)) -> dest_sink_serde(reqs_out);
+        source_stream_serde(reqs_in) -> map(Result::unwrap) -> map(|((client, req), _a): ((usize, SealedSetOfIndexedValues<Request>), _)| (client, req))
+          -> group_by(SSIV_BOT, ssiv_merge) -> [0]lookup_class;
+        source_iter(client_class) -> [1]lookup_class;
+        lookup_class = join()
+          -> map(|(client, (li, class))| ((client, class), li))
+          -> map(|m| (m, out_addr)) -> dest_sink_serde(out);
+          // -> for_each(|x| println!("{:?}\n", x));
```
Comment (MingweiSamuel) [inline hydroflow/examples/shopping/flows/server_state_flow.rs:45]: remove?

File: hydroflow/examples/shopping/misc/bp.pdf
Comment (MingweiSamuel) [inline hydroflow/examples/shopping/misc/bp.pdf:1]: remove

File: hydroflow/examples/shopping/misc/bpfl.pdf
Comment (MingweiSamuel) [inline hydroflow/examples/shopping/misc/bpfl.pdf:1]: remove

File: hydroflow/src/lang/lattice/bottom.rs
Comment (MingweiSamuel) [inline hydroflow/src/lang/lattice/bottom.rs:1]: remove

File: hydroflow/src/lang/lattice/mod.rs
Comment (MingweiSamuel) [inline hydroflow/src/lang/lattice/mod.rs:1]: remove

File: hydroflow/src/lang/lattice/qtofl.rs
Comment (MingweiSamuel) [inline hydroflow/src/lang/lattice/qtofl.rs:1]: remove?

File: hydroflow/src/lang/lattice/seal.rs
Comment (MingweiSamuel) [inline hydroflow/src/lang/lattice/seal.rs:1]: remove, equivalent to `Bottom<Fake<T>>`

File: hydroflow/src/lang/lattice/sealed_set_of_indexed_values.rs
Comment (MingweiSamuel) [inline hydroflow/src/lang/lattice/sealed_set_of_indexed_values.rs:1]: remove

File: hydroflow/src/lang/lattice/string_prefix.rs
Comment (MingweiSamuel) [inline hydroflow/src/lang/lattice/string_prefix.rs:1]: remove

File: hydroflow/src/lang/lattice/vec_prefix.rs
Comment (MingweiSamuel) [inline hydroflow/src/lang/lattice/vec_prefix.rs:1]: remove

File: lattices/src/test.rs
Comment (MingweiSamuel) [inline lattices/src/test.rs:1]: todo??

**Review by MingweiSamuel — COMMENTED** [review]
remove the dot files, pdf files, and hydroflow lattices

File: hydroflow/examples/lamport_clock/main.rs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    role: Role,
+    #[clap(long, value_parser = ipv4_resolve)]
+    addr: Option<SocketAddr>,
+    #[clap(long, value_parser = ipv4_resolve)]
+    server_addr: Option<SocketAddr>,
+    #[clap(value_enum, long)]
+    graph: Option<GraphType>,
+}
+
+#[tokio::main]
```
Comment (zzlk) [inline hydroflow/examples/lamport_clock/main.rs:38]: hydroflow::main

**Review by zzlk — APPROVED** [review]

**Pull request merged**